### ISSUE TITLE
Save Locks in Plugin to reuse

### ISSUE THREAD

**Issue by dmckeen** [unlock-protocol/unlock-wordpress-plugin#53]
Would be nice to be able to save your locks in the plugin with a friendly name so you can easily choose that lock when configuring the unlock block.  Bonus if we can group similar locks to choose them all at once.

**Comment by julien51** [comment]
That's a good point! We'll get that on the list of stuff for the next version!

**Comment by minderfolden** [comment]
Yeah, I have one lock for the top tier community members who get everything. They're always green-lit for every lock on the site. Then there are some of the "I'm trying this out" level who get some of the gated content. So I'm constantly having to go back to my notes to find the lock addresses to copy/paste them in. Would be great to have them as recents, or favorites, or something. Without any coding skills, I'd say this might be a good compromise between bare bones simple & having extra functionality without the complexity of having to use the whole membership site plugin that does **everything**.
